app/src/components/InputChoiceGroup.tsx

```diff
@@ -43,13 +45,16 @@ const InputChoiceGroup = (props: Props): ReactElement => {
           <input
             checked={choice.checked}
             className={`usa-${type}__input usa-${type}__input--tile`}
-            id={choice.value}
+            id={`${choice.name}-${choice.value}`}
```

Comment:
Notes for reviewers: The main change in this component is to make the `name` prop no longer optional because I wanted to create a unique html id for each input.

**Question:** I repeat the following a few times in this file because I am using a `.map((a) => (<jsx here>))`. What's the best way to DRY this up? 

```
{`${choice.name}-${choice.value}`}
```
Would it be to switch to something like this?

```
.map((x) => {
  const id = `${choice.name}-${choice.value}`
  return (
    <jsx here>
  )
}
```

Reply:
@aligg @sawyerh Thank you both!